home.blade.php

raniutami opened the pull request:
ini homepage udah pake bootstrap, kurang konten sama css nya masih perlu dimodif, logonya belum dimasukin

ilham77 commented:
branchnya jangan terpisah2 filenya... gue jadi ga bisa test branch ini karena routenya ada di branch lain.

raniutami committed:
update home

logo dan warna

raniutami committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into home-blade

raniutami committed:
update home

update footer, warna, dll

raniutami committed:
update home

update ukuran navbar, dan logo

ilham77 commented:
@raniutami kalau bisa css nya dibuat external, supaya yang lain stylenya sama

antoooks committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into login-sso

antoooks committed:
Revert "Integrasi dengan API SSO"

This reverts commit 7c4824bb8e42b11750cc141d19a1ef37f36798aa.

# Conflicts:
#	uilancer/resources/views/home.blade.php

antoooks committed:
Merge remote-tracking branch 'refs/remotes/origin/home-blade' into login-sso

# Conflicts:
#	uilancer/app/Http/routes.php
#	uilancer/composer.json
#	uilancer/composer.lock

raniutami commented:
@ilham77 iya tadinya juga mau digituin kok tapi belum sempet misahin, makasih sarannya :)

antoooks committed:
Merge pull request #10 from ilham77/login-sso

Login sso

raniutami committed:
update css

cssnya udah dipisah

raniutami committed:
update css

itsantifun committed:
login form

pembuatan login form dan menghubungkan halaman utama ke halaman login

itsantifun committed:
Merge pull request #11 from ilham77/login-official

login form

raniutami committed:
update

raniutami committed:
update fix

sudah dirapihin fix tampilannya bagian home dan login, css juga udah
dipisah, ada background gambar juga, warnanya juga udah diubah, isi
kontennya udah ditambah(gak ada yang lorem ipsum)

raniutami commented:
sudah jadi homepagenya sama yang login form udah dirapihin juga cssnya. cssnya juga udah dipisah filenya.

ilham77 commented:
@raniutami kok ga dimerge -_-

ilham77 commented:
@raniutami kok ga dimerge dengan master -_-
